core/git_command.py

```diff
@@ -31,6 +31,22 @@ class GitSavvyError(Exception):
     pass
 
 
+class IsolatedGitCommand(GitCommand):
+    """
+    Allow accessing GitCommand outside the scope of a WindowCommand or
+    TextCommand.  You must manually pass in a valid repository path.
+    """
+    def __init__(self, repo_path=None):
+        self._repo_path = repo_path
+
+    def git(self, *args):
```

Comment:
I thought it was allowing me to call git with fewer options but now seeing that the calls are both *args that doesn't seem to be the case.